<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cMainWindow</class>
 <widget class="QMainWindow" name="cMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="m_splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="m_buttonLayout">
          <item>
           <widget class="QToolButton" name="m_listAdd">
            <property name="text">
             <string/>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonIconOnly</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="m_listEdit">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="m_listRemove">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QToolBox" name="m_toolBox">
          <property name="currentIndex">
           <number>2</number>
          </property>
          <widget class="QWidget" name="m_manufacturerListTab">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>89</width>
             <height>362</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Manufacturer</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QTreeView" name="m_manufacturerList">
              <property name="contextMenuPolicy">
               <enum>Qt::CustomContextMenu</enum>
              </property>
              <property name="editTriggers">
               <set>QAbstractItemView::EditKeyPressed</set>
              </property>
              <property name="rootIsDecorated">
               <bool>false</bool>
              </property>
              <attribute name="headerVisible">
               <bool>false</bool>
              </attribute>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="m_distributorListTab">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>89</width>
             <height>362</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Distributor</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QTreeView" name="m_distributorList">
              <property name="contextMenuPolicy">
               <enum>Qt::CustomContextMenu</enum>
              </property>
              <property name="editTriggers">
               <set>QAbstractItemView::EditKeyPressed</set>
              </property>
              <property name="rootIsDecorated">
               <bool>false</bool>
              </property>
              <property name="headerHidden">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="m_storage">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>89</width>
             <height>362</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Storage</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <widget class="QTreeView" name="m_storageList">
              <property name="rootIsDecorated">
               <bool>true</bool>
              </property>
              <attribute name="headerVisible">
               <bool>false</bool>
              </attribute>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="m_partListTab">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>89</width>
             <height>362</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Part</string>
           </attribute>
          </widget>
          <widget class="QWidget" name="m_projectListTab">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>89</width>
             <height>362</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Project</string>
           </attribute>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget_2">
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,99">
        <item>
         <widget class="QTabWidget" name="m_mainTab">
          <property name="currentIndex">
           <number>-1</number>
          </property>
          <property name="documentMode">
           <bool>true</bool>
          </property>
          <property name="tabsClosable">
           <bool>true</bool>
          </property>
          <property name="movable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QMdiArea" name="m_mdiArea"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="m_menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="m_statusBar"/>
  <widget class="QToolBar" name="m_toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
